tools/add_triage_signature.py

```diff
@@ -437,13 +469,68 @@ def _update_ticket(self, url, issue_key, should_update=False):
             self._update_triaging_ticket(issue_key, report, should_update=should_update)
 
 
+class IOErrorsSignature(Signature):
+    def __init__(self, jira_client):
+        super().__init__(jira_client, comment_identifying_string="h1. I/O errors")
+
+    def _update_ticket(self, url, issue_key, should_update=False):
+        url = self._logs_url_to_api(url)
+
+        try:
+            md = get_metadata_json(url)
+        except Exception as e:
+            logger.error("Error getting metadata for %s at %s: %s, it may have been deleted", issue_key, url, e)
+            return
+
+        cluster = md['cluster']
+        cluster_id = cluster['id']
+
+        with NamedTemporaryFile() as temp:
```

Comment:
Maybe we can manage with only the temp dir defined next?
To make it easy on the indentation level